actions/release-secrets/README.md

```diff
@@ -0,0 +1,18 @@
+# Release Secrets
+
+This action can be used for getting release secrets from SSM.
+
+# Requirements
+
+The repository must be configured with OIDC allowing access to an AWS account.
+
+# Example
+
+This example uses the release-secrets action to get an NPM token.
+```
+- uses: launchdarkly/gh-actions/actions/release-secrets
```

Comment:
I am reasonably sure this is what the path will look like. But I can change it if I find out that isn't the case.